FJP starvation spec

johanandren opened the pull request:
Refs #26973

johanandren committed:
Doesn't fix the fail but takes test time down

from 3m to 15s or so

johanandren commented on akka-actor-tests/src/test/scala/akka/dispatch/ForkJoinPoolStarvationSpec.scala:
```diff
@@ -10,8 +10,8 @@ import com.typesafe.config.ConfigFactory
 
 object ForkJoinPoolStarvationSpec {
   val config = ConfigFactory.parseString("""
+      |akka.actor.serialize-messages = off
```
This took a lot of time. Doesn't explain why the test started failing when we switched FJP but should probably be done anyway.

helena replied:
Do you know why it was there in the first place?

johanandren replied:
We have it on in pretty much all test suites for some historical reason, to catch messages that aren't serializable I guess.

johanandren committed:
Lower throughput to default

This gives more dispatched tasks that the external ones can be sandwiched between
which should stop the test from failing even though the JDK 11 FJP is less fair.

patriknw reviewed (approved):
LGTM, let's merge this first and follow up with other improvements separately
